pkg/meta/dapp.go

```diff
@@ -16,8 +16,8 @@ type NodePort struct {
 
 // SidecarPort represents the port for communication between node and sidecar
 type SidecarPort struct {
-	Read  int `json:"read"`
-	Write int `json:"write"`
+	LBRead  int `json:"read"`
+	LBWrite int `json:"write"`
```

Comment:
Does this have any impact? I mean, the json is not changed, and there is nothing else that this could be referencing.

Reply:
I'll change the json. I think it's better to specify that these variables are used only in the LB Sidecar, and not also in the broker-specific ones.